libraries/botbuilder-dialogs-adaptive-runtime-integration-azure-functions/package.json

```diff
@@ -33,7 +33,7 @@
     "botbuilder-dialogs-adaptive-runtime": "4.1.6",
     "botbuilder-dialogs-adaptive-runtime-core": "4.1.6",
     "mime": "^2.5.2",
-    "zod": "~1.11.17"
+    "zod": "^3.0.0"
```

Comment:
Use the latest `^3.22.4` https://www.npmjs.com/package/zod?activeTab=versions, also in other packages as well.

Reply:
Remember to update it in the description as well